reference/7.0/Microsoft.PowerShell.Core/About/about_Calculated_Properties.md

```diff
@@ -0,0 +1,506 @@
+  - `ascending`/`descending` - optional
+
+For more information, see the [Examples](#examples) section of this article.
+
+## Attribute definitions
+
+- `name`/`label` - Specifies the name of the property being created. You can
+  use `name` or its alias, `label`, interchangeably.
+- `expression` - A script block used to calculate the value of the new property.
+- `alignment` - Used by cmdlets that produce tabular output to define how the
+  values is displayed in a column. The value must be `'left'`, `'center'`, or
+  `'right'`.
+- `formatstring` - Specifies a format string that defines how the value is
+  formatted for output. For more information about format strings, see
+  [Format types in .NET](/dotnet/standard/base-types/formatting-types).
+- `width` - Specifies the maximum width column in a table when the value is
+  displayed. The value must be greater than `0`.
+- `depth` - The **Depth** parameter of `Format-Custom` specifies the depth of
+  expansion for all properties. The `depth` attribute allows you to specify the
+  depth of expansion per property.
+- `ascending` / `descending` - Allows you to specify the order of sorting for
+  one or more properties. These are boolean values.
+
+The hashtable keys need not be spelled out as long as the specified name prefix
+is unambiguous. For example, `n` can be used in lieu of `Name` and `e` can be
+used in lieu of `Expression`.
+
+## Examples
+
+### Compare-Object
+
+With calculated properties, you can control how the properties of the input
+objects are compared.
+
+```powershell
+Compare-Object @{p=1} @{p=2} -property @{ Expression = { $_.p % 2 } }
+```
+
+```Output
+ $_.p % 2  SideIndicator
+---------- -------------
+         0 =>
+         1 <=
+```
+
+### ConvertTo-Html
+
+`ConvertTo-Html` can convert a collection of objects to an HTML table.
+Calculated properties allow you to control how the table is presented.
+
+```powershell
+Get-Alias |
+  ConvertTo-Html Name,
+                 Definition,
+                 @{n='ParamCount';e={$_.Parameters.Keys.Count};align='center'} |
+    Out-File .\aliases.htm -Force
+```
+
+This example creates an HTML table contain a list of PowerShell aliases and the
+number parameters for each aliased command. The values of **ParamCount** column
+are centered.
+
+### Format-Custom
+
+`Format-Custom` provides a custom view of an object in a format similar to a
+class definition. More complex objects can contain members that are deeply
+nested with complex types. The **Depth** parameter of `Format-Custom` specifies
+the depth of expansion for all properties. The `depth` attribute allows you to
+specify the depth of expansion per property.
+
+In this example we use the `depth` attribute to simplify the custom output for
+the `Get-Date` cmdlet . `Get-Date` returns a **DateTime** object. The **Date**
+property of this object is also a **DateTime** object, so the object is nested.
+
+```powershell
+Get-Date | Format-Custom @{e={$_.Date};depth=1},TimeOfDay
```

Comment:
I know it adds some length but we should avoid aliases in code samples as much as possible?

Reply:
I chose to use short form to two reasons:
1. to demonstrate the short form
2. limited column width of About_ topics